FinSearchUnified/Tests/Helper/StaticHelperTest.php

```diff
@@ -388,27 +513,6 @@ public function testUseShopSearchForBackendRequests()
         // Assign mocked session variable to application container
         Shopware()->Container()->set('front', $front);
 
-        // Create Mock object for Shopware Config
```

Comment:
Why was this code block removed? Maybe this code now runs because it fails somewhere else and does not test what it should actually test?

Reply:
It was removed because it did not have any significant impact on the test outcome, since we are testing for the backend request only in this case.